### DISCUSSION TITLE
How to use own detection model like YOLOV8

### DISCUSSION THREAD

**Discussion by nikhilgarg1210** [felixdittrich92/OnnxTR#55] [Q&A]
### Bug description

Hi @felixdittrich92, i want to use yolov8 detection model instead of the prebuilt models by you. if yes how can i use them. could you please help me.

### Code snippet to reproduce the bug

```python
# Load the ONNXTR OCR model
def load_onnxtr_model():
    return ocr_predictor(
        det_arch='db_resnet50', #i need to use yolov8
        reco_arch='crnn_vgg16_bn',
        detect_orientation=True,
        straighten_pages=True,
        assume_straight_pages=False,
    )
```

### Error traceback

```nothing```

### Environment

python3

**Comment by felixdittrich92** [comment]
Hi @nikhilgarg1210 :wave:;

Currently it's not supported to load a yolo exported model for text detection :)

Do you need only the recognition part from OnnxTR or also the rest like image straighten and the `DocumentBuilder` features ?

We have an contribution module where you can load an Yolo Onnx exported model (no OBB yet) https://mindee.github.io/doctr/using_doctr/using_contrib_modules.html
but that was more meant for artefact detection models ^^
